app/serializers/post_serializer.rb

```diff
@@ -212,6 +213,14 @@ def user_title
     object&.user&.title
   end
 
+  def title_is_group
+    object&.user&.title == object.user&.primary_group&.title
```

Comment:
why do we object&.user on one side an object.user on the other ?

Reply:
Whoops, copy-paste error. Probably harmless, so I'm going to submit as is.